docs/pages/connect-your-client/tsh.mdx

```diff
@@ -917,24 +921,43 @@ c0a02222-70dc-4be8-9308-7b7901d2d600 kube alice                  Nov 26 20:36:16
 d0a04442-70dc-4be8-9308-7b7901d2d600 ssh  navin        test      Nov 26 16:36:16 UTC
 ```
 
-To play sessions, use the `tsh play <sessionid>` command. The screen will reset playing the interactive session with the timestamp in the upper right.
+### Playing recordings
+
+To play a session recording, run the `tsh play` command with the ID of a session
+as returned by `tsh recordings ls`: 
 
 ```code
 $ tsh play c0a02222-70dc-4be8-9308-7b7901d2d600
-root@ubuntu:/# df                                                                     2022-11-26T21:20:52.715Z
-Filesystem     1K-blocks    Used Available Use% Mounted on
-overlay         20959212 4387284  16571928  21% /
-tmpfs              65536       0     65536   0% /dev
-tmpfs            1982720       0   1982720   0% /sys/fs/cgroup
-/dev/nvme0n1p1  20959212 4387284  16571928  21% /etc/hosts
-shm                65536       0     65536   0% /dev/shm
-tmpfs            3410436      12   3410424   1% /run/secrets/kubernetes.io/serviceaccount
-tmpfs            1982720       0   1982720   0% /proc/acpi
-tmpfs            1982720       0   1982720   0% /sys/firmware
-root@ubuntu:/# exit
-exit
 ```
 
+Users of self-hosted Teleport clusters can also run `tsh play` with the path to
```

Comment:
With cloud and external audit storage it's also reasonable to expect that cloud customers can play from a file.